java/server/test/org/openqa/selenium/grid/node/NodeTest.java

```diff
@@ -424,6 +424,17 @@ private File getTemporaryFilesystemBaseDir(TemporaryFilesystem tempFS) {
     return baseDir;
   }
 
+  //Test that the draining command sets Host status to DRAINING
+  @Test
+  public void drainingNodeStatusDraining() {
+
+  }
```

Comment:
Is this only to set the node to draining? 🤔 

I think we can do that inside `Before` or something?

Reply:
Being taken care by @mmerrell